### ISSUE TITLE
Index newly modified records for exhibits

### ISSUE THREAD

**Issue by afred** [WGBH/AAPB2#766]
**Done when** the following exhibits are re-ingested
  * civil rights
  * climate change
  * station histories

**Comment by afred** [comment]
@caseyedavis12 are we waiting on a green-light from you before doing this?

cc @mccalluc

**Comment by caseyedavis12** [comment]
@afred: yes, this will need to happen on Monday if that's ok. I plan to updating all records tomorrow.


If not, I can try to fit it in today.

**Comment by afred** [comment]
@caseyedavis12 monday works. Please confirm here whenever it's ready

**Comment by caseyedavis12** [comment]
Closing in favor of #783
